pallets/gear/src/manager.rs

```diff
@@ -80,9 +80,9 @@ where
                 data.into_iter().map(|(k, v)| (k.into(), v)).collect()
             };
             let _ = actor.program.set_pages(pages_data);
-            (id, actor)
+            (id, Some(actor))
```

Comment:
for pallet there is not always some case.
in storage we have enum `Program` (located at `gear/common/lib.rs`).
`Program::Terminated` is `None` case for `CollectState` here

Reply:
done